Sources/dump-unified-graph/DumpUnifiedGraph.swift

```diff
@@ -0,0 +1,109 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import ArgumentParser
+import SymbolKit
+
+@main
+struct DumpUnifiedGraph: ParsableCommand {
+    @Option(help: ArgumentHelp("module's symbol graph to output", discussion: "will infer a single module, but will fail if multiple modules are being loaded"))
+    var moduleName: String?
+
+    @Flag(inversion: .prefixedNo,
+          help: "whether to pretty-print the output JSON (default: true)")
```

Comment:
ArgumentParser actually adds the `(default:` text by default (🙃) so it's being emitted twice currently:

```
  --pretty-print/--no-pretty-print
                          whether to pretty-print the output JSON (default: true) (default: true)
```

```suggestion
          help: "whether to pretty-print the output JSON")
```